MSAL/src/public/MSALWebviewParameters.h

```diff
@@ -94,6 +95,13 @@ NS_ASSUME_NONNULL_BEGIN
 */
 - (nonnull instancetype)initWithAuthPresentationViewController:(MSALViewController *)parentViewController;
 
+
+#if !MSID_EXCLUDE_WEBKIT
```

Comment:
This is internal ifdef and is not applicable to MSAL. Let's remove it from here :)

Reply:
I see. Thank you for letting me know.